src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStat.tsx

```diff
@@ -73,6 +73,11 @@ const KeyStat = ({
               <Details
                 summary={summary?.dataDefinitionTitle[0] || 'Help'}
                 className={styles.definition}
+                visuallyHiddenText={
+                  summary?.dataDefinitionTitle[0]
+                    ? ` help for ${keyStat.title}`
+                    : ''
+                }
```

Comment:
I think this is the wrong way round, this will only show the help text when the dataDefinitionTitle is set and just 'Help' otherwise.
Also it should just be `for ${keyStat.title}` as 'Help' is already in the summary and this will be read out with it ('Help help for whatevs').

You could also put the space in the Details component as it's likely to get missed if it has to be passed in with the text.